nuxeo-core/nuxeo-core-storage-dbs/src/main/java/org/nuxeo/ecm/core/storage/dbs/DBSTransactionState.java

```diff
@@ -1186,7 +1186,7 @@ protected void findDirtyPaths(List<?> value, String path) {
         }
 
         protected void findDirtyPaths(ListDiff value, String path) {
-            String newPath = path + "/*";
+            String newPath = path;
```

Comment:
Out of curiosity, was there a bug before that wasn't covered by any test? This looks like a more general issue.

Reply:
yes from point of view it's a general issue. i think before the development of https://jira.nuxeo.com/browse/NXP-26669 we didn't have a test that cover the using of the `relatedText` of document in full text search and when i did NXP-26669 i didn't launch a T&P with a mongo profile. I asked Florent on the platform dev to get lore feedbacks

Reply:
OK, so it really seems unrelated to NXP-28406.
I think it deserves a separate commit with NXP-26669, or even better a dedicated JIRA issue? (but then it would need a dedicated unit test, so if running a T&P with MongoDB fails once NXP-26669 has been merged, then using this issue in the comment should be OK).